clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2841,23 +2838,22 @@ class KernelCallOperatorVisitor
     }
     return true;
   }
+
+  CXXMethodDecl *GetCallOperator() {
```

Comment:
```suggestion
  CXXMethodDecl *getCallOperator() {
```
Likewise for visitCallExpr(...).

Reply:
> Likewise for visitCallExpr(...).

@premanandrao FYI, it appears that we need to name Visitor functions starting with upper case. Otherwise, the Visitor functions in the Visitor's stack/list do not get called as part of the Visitor traversal.
Example,  when we change `VisitCallExpr()` to `visitCallExpr(...)`, we never reach `VisitCallExpr(...)` during the visitor traversal.

Reply:
Makes sense.   We can fix that all together some other time.